apis/bulkupload/bulkserviceapi.yaml

```diff
@@ -57,397 +74,903 @@ paths:
+                $ref: '#/components/schemas/Success-UploadUsersinBulk'
+              example:
+                id: api.user.upload
+                ver: v1
+                ts: 2020-12-01 16:24:35:250+0000
+                params:
+                  resmsgid: 
+                  msgid: bf2d6e695e927a885eaae649c45b84b4
+                  err: 
+                  status: success
+                  errmsg: 
+                responseCode: OK
+                result:
+                  processId: 01316323485186457631
         400:
-          description: BAD REQUEST. The "bulk upload user" operation failed ! The
+          description: 'BAD REQUEST. The "bulk upload user" operation failed ! The
             possible reason for failure is that you may have missed providing input
-            for a mandatory parameter.
+            for a mandatory parameter.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UploadOrganisationsinBulk-BadRequest'
+              example:
+                id: api.user.upload
+                ver: v1
+                ts: 2020-12-07 05:55:54:636+0000
+                params:
+                  resmsgid: 
+                  msgid: cfaddf7e346675b6aeeb9803e32d0ed8
+                  err: INVALID_COLUMNS
+                  status: INVALID_COLUMNS
+                  errmsg: 'Invalid column: firstnamee. Valid columns are: firstname, firstName, roles, roles, usertype, userType, user id, userId, school external id, orgExternalId, userid, userId, orgid, orgId, user_type, userType, school id, orgId, phone, phone, orgexternalid, orgExternalId, mobile phone, phone, name, firstName, email, email.'
+                responseCode: CLIENT_ERROR
+                result: {}
+        401:
+          description: Unauthorized
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BulkUploadErrorResponse'
-        500:
-          description: INTERNAL SERVER ERROR ! Looks like something went wrong! These
-            errors are tracked automatically, but if the problem persists feel free
-            to contact us. In the meantime, try refreshing.
-          content: {}
+                $ref: '#/components/schemas/UploadOrganisationsinBulk-BadRequest'
+              example:
+                id: api.user.upload
+                ver: v1
+                ts: 2020-12-07 05:44:39:138+0000
+                params:
+                  resmsgid: 
+                  msgid: 1b4f72063d1b43ef397e9b6ded1dbd90
+                  err: UNAUTHORIZED_USER
+                  status: SERVER_ERROR
+                  errmsg: You are not authorized.
+                responseCode: CLIENT_ERROR
+                result: {}
+      deprecated: false
+      security: []
   /user/v2/bulk/upload:
     post:
       tags:
-      - Bulk Operation APIs
+      - upload shadow users and self-declared users
       summary: Upload Shadow Users in Bulk
```

Comment:
@Hari-stackroute could you please check if this API still needs to be there or if it is deprecated? The concept of 'Shadow DB' is no longer a valid concept. Hence the question.

Reply:
@Sunitashelar the code for this still there and some part of the same api works for processing self-declared users with bulk-upload. Once after refactoring the code we will alter the description. So as of now let it be like this.

Reply:
OK